source/UberRides/RideRequestButton.swift

```diff
@@ -105,25 +105,31 @@ import CoreLocation
         sizeToFit()
     }
     
-    // add title, image, and sizing configuration
     override func setContent() {
         super.setContent()
         
+        setSizesConfigs()
+        addTitle()
+        addImage()
+    }
+    
+    private func setSizesConfigs() {
```

Comment:
I would combine this with `addTitle()` since it's really just setting up the `uberTitleLabel`